packages/datasource-elasticsearch/package.json

```diff
@@ -27,7 +23,13 @@
     "test": "docker-compose up -d; sleep 5 && jest",
     "publish:package": "semantic-release"
   },
+  "peerDependencies": {
+    "@forestadmin/datasource-toolkit": "~1.29.0",
```

Comment:
It will only allow patches. Let see if we need to change this to carret ^ in the future.